.gitignore

```
@@ -1,3 +1,5 @@
+
 .idea/*
 *.iml
 target/*
+/logs/app.log
```

```suggestion
logs/*
```
let's ignore all files in logs folder